### ISSUE TITLE
MacOS support

### ISSUE THREAD

**Issue by Kataiser** [Kataiser/tf2-rich-presence#53]
I don't own a Mac, but I'd like this project to work on MacOS anyway.

**Comment by Jan200101** [comment]
I'll give it a go
Shouldn't be too hard considering python is Crossplatform

**Comment by Jan200101** [comment]
alright this is gonna take WAY longer
the entire codebase is a mess and not optimized for crossplatform usage
not to mention the dependency on windows executables

**Comment by Kataiser** [comment]
Yeah, the code (especially main.py) is a mess due to me not really expecting this project to grow much, and then not refactoring enough. Thanks for at least giving it an attempt, though!

**Comment by Jan200101** [comment]
got it to work on a crossplatform base (like mentioned in the Linux issue)
the only thing missing right now is the names of the processes
